aws/rhel-8.5-nightly-x86_64/main.tf

```
@@ -1,7 +1,7 @@
 module "aws" {
   source = "../_base"
 
-  name             = "rhel-8.5-devel-x86_64"
+  name             = "rhel-8.5-nighlty-x86_64"
```

Just one more thing, sorry. :/

```suggestion
  name             = "rhel-8.5-nightly-x86_64"
```

---

Ah crap. Thanks.